<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: qlineargradient(spread:pad, x1:0.498, y1:1, x2:0.495673, y2:0.00028853, stop:0 rgba(255, 129, 0, 255), stop:1 rgba(255, 255, 255, 255))}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>581</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
background-color: rgb(248, 255, 243);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>581</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border 0px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="QTableWidget" name="tableWidget_procesos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>451</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	background-color: rgba(125, 151, 255, 0.15);
	border-radius: 15px;
	border: 1px solid rgba(125, 151, 255, 0.15);
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ejecutar">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>60</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	background-color: rgba(125, 151, 255, 0.15);
	border-radius: 15px;
	border: 1px solid rgba(125, 151, 255, 0.15);
}</string>
       </property>
       <property name="text">
        <string>Ejecutar</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(0,0,0);
}</string>
      </property>
      <property name="text">
       <string>Resultado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(0, 0, 0);
}</string>
     </property>
     <property name="text">
      <string>Selección de Algoritmo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_fcfs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>FCFS</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_sjf">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SJF</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_rr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Round Robin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_priority">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prioridades</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(0,0,0);
}</string>
     </property>
     <property name="text">
      <string>Configuración del Algoritmo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_config">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgba(255, 255, 255,0.45);
	border-radius: 15px;
	border: 1px solid rgba(255, 255, 255,0.15);
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="roundRobinPage">
      <widget class="QWidget" name="roundRobinWidget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>101</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Quantum</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="quantumInput">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>561</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgba(255, 255, 255, 0.45);
	border-radius: 15px;
	border: 1px solid rgba(255, 255, 255, 0.15);
}
	</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QTableWidget" name="tableWidget_resultados">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>461</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgba(125, 151, 255, 0.15);
	border-radius: 15px;
	border: 1px solid rgba(125, 151, 255, 0.15);
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgba(125, 151, 255, 0.15);
	border-radius: 15px;
	border: 1px solid rgba(125, 151, 255, 0.15);
}</string>
      </property>
      <property name="text">
       <string>T.P</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border: 1px solid rgb(0,0,0);
}</string>
     </property>
     <property name="text">
      <string>Proyecto de Sistemas Operativos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>581</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(212, 249, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
    <zorder>frame_3</zorder>
    <zorder>frame_2</zorder>
    <zorder>label_3</zorder>
    <zorder>radioButton_fcfs</zorder>
    <zorder>radioButton_sjf</zorder>
    <zorder>radioButton_rr</zorder>
    <zorder>radioButton_priority</zorder>
    <zorder>label_4</zorder>
    <zorder>stackedWidget_config</zorder>
    <zorder>frame_7</zorder>
    <zorder>label_8</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProyecto_de_Sistemas_Operativos">
    <property name="title">
     <string>Proyecto de Sistemas Operativos</string>
    </property>
   </widget>
   <addaction name="menuProyecto_de_Sistemas_Operativos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
